Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6-alpine
+RUN mkdir /app
+WORKDIR /app
+ADD requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
AFAIR, it is recommended to write `pip install ...` command with `--no-cache-dir` option as a good Docker practice.
It's not essential by now as we have no requirements at the moment, but you can add this option in the future.

Reply:
> AFAIR, it is recommended to write `pip install ...` command with `--no-cache-dir` option as a good Docker practice.
> It's not essential by now as we have no requirements at the moment, but you can add this option in the future.

Well, I think now it's a right time to do this.